season/the_basics.md

```diff
@@ -158,5 +161,10 @@ end
 
 **Answer 12:**
 ```ruby
-# write your code here
+  def caps(string)
+    puts "#{string}".upcase
+
+  end
+
```

Comment:
Nitpick: you can lose that blank line on 166; it's not necessary here.

On 165, there's no need to interpolate the string - it's already a string (so you've got a kind of string inception there 😄). You can simply put `string.upcase`.